spec/rspec/mocks/matchers/receive_spec.rb

```diff
@@ -236,6 +236,12 @@ def receiver.method_missing(*); end # a poor man's stub...
         end
 
         context "on a frozen object" do
+          before do
+            if RUBY_VERSION < "2.2" && RUBY_VERSION >= "2.0" && RSpec::Support::Ruby.mri?
```

Comment:
Typically, we add [such checks to `rspec-support`](https://github.com/rspec/rspec-support/blob/main/lib/rspec/support/ruby_features.rb#L79), but I'm fine with keeping this check here as it's for specs only.